### PR TITLE
Feature/query and update tbl

### PR DESCRIPTION
**Pull request by Benawi**
### General requirements
- [x] Make sure that you used the correct Gitflow.
- [x] Make sure that you documented your work in a professional way.
### Project requirements

- [x] Populate data to the table
- [x] Transactions allow making changes in isolation and rolling back if needed
- [x] Demonstrates BEGIN TRANSACTION, COMMIT, and ROLLBACK
- [x] Allows verifying changes before committing
- [x] Demonstrates deleting subsets of rows
- [x] Creates a savepoint within a transaction
- [x] Allows rolling back to a mid-transaction savepoint
- [x] Basic aggregation queries like COUNT, AVG, MIN, and MAX functions
- [x] Conditional filtering WHERE clause to filter result sets, group results by a column like animal type and apply aggregates per group
- [x] All queries tested and works all [Screenshot 1](https://github.com/Benawi/Vet-clinic-database/assets/21217148/27dfcaee-3d1d-4864-94fe-1fc7e353f993), [Screenshot 2](https://github.com/Benawi/Vet-clinic-database/assets/21217148/22b7012a-99f4-4ae9-9dc2-0ecae16c4a2f), [Screenshot 3](https://github.com/Benawi/Vet-clinic-database/assets/21217148/73b03944-a13a-4b40-81bd-aab8269c82df), [Screenshot 4](https://github.com/Benawi/Vet-clinic-database/assets/21217148/72a66819-f422-4ae9-8d59-d5a466e72009), [Screenshot 6](https://github.com/Benawi/Vet-clinic-database/assets/21217148/b318c555-2f08-4655-b624-275cacd0385e), [Screenshot 7](https://github.com/Benawi/Vet-clinic-database/assets/21217148/cb338473-9301-4169-9f7d-b0027b2c13d0), [Screenshot 8](https://github.com/Benawi/Vet-clinic-database/assets/21217148/c375eff6-11da-4c5e-8a21-0387385deeda)

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -5,3 +5,20 @@ INSERT INTO animals VALUES
...
+(default, 'Squirtle', '19930402', 3, false, 12.13 ),
+/* Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once. */
+(default, 'Angemon', '20050712', 1, true, -45 ),
+/* Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times. */
+(default, 'Boarmon', '20050707', 7, true, 20.4 ),
+/* Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times. */
+(default, 'Blossom', '19981013', 3, true, 17 ),
+/*  Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times.*/
+(default, 'Ditto', '20220614', 4, true, 22 );
+(default, 'Ditto', '20220614', 4, true, 22 );
```
Comment (ShadyShawkat) [inline data.sql]: - [ ] Kindly make sure to insert the given data correctly. Some of the given data's `weights` are in negative (the first four records), as you can see from the following image, please make sure to correct these values accordingly.

image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Benawi, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Benawi** [comment]
Changes ♻️
- [x] Input data inserted correctly

**Review by gemmen29 — APPROVED** [review]
Hi @Benawi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**